website/source/docs/config/index.html.md

```diff
@@ -111,6 +111,22 @@ For Consul, the following options are supported:
 
   * `token` (optional) - An access token to use to write data to Consul.
 
+  * `tls_ca_file` (optional, mandatory when scheme is https) - The path to the CA certificate
```

Comment:
In many cases, there verification is only done in one direction. For example, you browser verifies the authenticity of the web server, but the web server does not care about the identity of the client. Similarly, you can do a one way authentication "is consul who it claims to be" without using a `cert` or `key`. Alternatively, Consul may verify the client identity which requires those. In that case the verification is done in both directions and is "mutual".